### PR TITLE
Adding OrderBy Query Option to Chat.GetMessages Requests

### PR DESCRIPTION
**Pull request by evanypark**
Updating Documentations/Change logs for Enabling new Query Option, OrderBy, when handling Chat.GetMessages Requests.
Users will be allowed to sort chat messages in order of LastModifiedDateTime or CreatedDateTime.

### CODE REVIEW

File: api-reference/beta/api/chat-list-messages.md [hunk 1]
```diff
@@ -43,7 +43,11 @@ GET /chats/{chat-id}/messages
 
 ## Optional query parameters
 
-You can use the [$top](/graph/query-parameters#top-parameter) query parameter to control the number of items per response. Maximum allowed `$top` value is 50.
+| Name      | Description          |
+|:----------|:--------|
+| [$top](/graph/query-parameters#top-parameter)      | Control the number of items per response. Maximum allowed `$top` value is 50.  |
```
Comment (sumanac) [inline api-reference/beta/api/chat-list-messages.md:48]: Controls?

**Comment by eddie-lee-msft** [comment]
Can you add an example (or two) of using the new OrderBy query option?

File: api-reference/beta/api/chat-list-messages.md [hunk 2]
```diff
@@ -43,7 +43,11 @@ GET /chats/{chat-id}/messages
 
 ## Optional query parameters
 
-You can use the [$top](/graph/query-parameters#top-parameter) query parameter to control the number of items per response. Maximum allowed `$top` value is 50.
+| Name      | Description          |
+|:----------|:--------|
+| [$top](/graph/query-parameters#top-parameter)      | Controls the number of items per response. Maximum allowed `$top` value is 50.  |
+| [$orderBy](/graph/query-parameters#orderBy)  | Currently supports **LastModifiedDateTime** and **CreatedDateTime**.|
```
Comment (eddie-lee-msft) [inline api-reference/beta/api/chat-list-messages.md:49]: Can you add an example (or two) of using the new OrderBy query option? Here and in the v1.0 doc.

**Review by eddie-lee-msft — CHANGES_REQUESTED** [review]

File: api-reference/v1.0/api/chat-list-messages.md [hunk 3]
```diff
@@ -39,7 +39,11 @@ GET /chats/{chat-id}/messages
 
 ## Optional query parameters
 
-You can use the [$top](/graph/query-parameters#top-parameter) query parameter to control the number of items per response. Maximum allowed `$top` value is 50.
+| Name      | Description          |
+|:----------|:--------|
+| [$top](/graph/query-parameters#top-parameter)      | Controls the number of items per response. Maximum allowed `$top` value is 50.  |
+| [$orderBy](/graph/query-parameters#orderBy)  | Currently supports **LastModifiedDateTime** and **CreatedDateTime**.|
+
```
Comment (RamjotSingh) [inline api-reference/v1.0/api/chat-list-messages.md:46]: Let's remove v1 for now. Let's first ensure API is stable in beta and then move it to v1

**Review by RamjotSingh — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/chat-list-messages.md (hunk 2 above)
Comment (evanypark) [reply api-reference/beta/api/chat-list-messages.md:49]: Added orderBy query parameter to the sample below

**Review by eddie-lee-msft — APPROVED** [review]

File: changelog/Microsoft.Teams.Core.json [hunk 4]
```diff
@@ -2,7 +2,25 @@
 
   "changelog": [
     {
-      "ChangeList":[
+      "ChangeList": [
+        {
+          "Id": "ebf148a1-9b8d-4c5b-80db-8476f0aa6264",
+          "ApiChange": "Property",
+          "ChangedApiName": "OrderBy",
```
Comment (RamjotSingh) [inline changelog/Microsoft.Teams.Core.json:9]: ChangedApiName is prob going to be chatMessage and ApiChange is not property def. @Lauragra might be able to suggest correct stuff to put here.

**Review by RamjotSingh — APPROVED** [review]

File: api-reference/beta/api/chat-list-messages.md [hunk 5]
```diff
@@ -43,7 +43,11 @@ GET /chats/{chat-id}/messages
 
 ## Optional query parameters
 
-You can use the [$top](/graph/query-parameters#top-parameter) query parameter to control the number of items per response. Maximum allowed `$top` value is 50.
```
Comment (Lauragra) [inline api-reference/beta/api/chat-list-messages.md:46]: Please add an introductory sentence here: "This method supports the following [OData query parameters](/graph/query-parameters)."

File: api-reference/beta/api/chat-list-messages.md [hunk 6]
```diff
@@ -73,7 +79,7 @@ The following is an example of the request. `$top=2` is passed to retrieve two m
   "name": "get_allchatmessages_1"
 }-->
 ```msgraph-interactive
-GET https://graph.microsoft.com/beta/chats/19:2da4c29f6d7041eca70b638b43d45437@thread.v2/messages?$top=2
+GET https://graph.microsoft.com/v1.0/chats/19:2da4c29f6d7041eca70b638b43d45437@thread.v2/messages?$top=2&$orderBy=createdDateTime
```
Comment (Lauragra) [inline api-reference/beta/api/chat-list-messages.md:82]: Why did we change the example from beta to v1.0? This topic is in the beta folder so it should be using /beta in the query URL.

File: changelog/Microsoft.Teams.Core.json [hunk 7]
```diff
@@ -2,7 +2,25 @@
 
   "changelog": [
     {
-      "ChangeList":[
+      "ChangeList": [
+        {
+          "Id": "ebf148a1-9b8d-4c5b-80db-8476f0aa6264",
+          "ApiChange": "queryOption",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:8]: For this value, I think you want to use "Parameter".

File: changelog/Microsoft.Teams.Core.json [hunk 8]
```diff
@@ -2,7 +2,25 @@
...
+          "ChangedApiName": "chatMessage",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:9]: This would be the verb, in this case LIST.

File: changelog/Microsoft.Teams.Core.json [hunk 9]
```diff
@@ -2,7 +2,25 @@
...
+          "ChangeType": "Addition",
+          "Description": "OrderBy query option has been enabled to allow users to choose sorting order between LastModifiedDateTime and CreatedDateTime when retreiving chat messages.",
+          "Target": "chat"
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:12]: The target is chatMessage.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**